config/protocol_config.go

```diff
@@ -31,6 +31,9 @@ type ProtocolConfig struct {
 	Ip     string      `yaml:"ip"  json:"ip,omitempty" property:"ip"`
 	Port   string      `default:"20000" yaml:"port" json:"port,omitempty" property:"port"`
 	Params interface{} `yaml:"params" json:"params,omitempty" property:"params"`
+
+	MaxServerSendMsgSize string `yaml:"max-server-send-msg-size" json:"max-server-send-msg-size,omitempty"`
```

Comment:
Please add a comment for its unit, like "1mib=1024b, 1mb=1000b", or  "Please refer to go-humanize package for its unit.".

Reply:
okay